lincheck/src/main/java/org/jetbrains/kotlinx/lincheck/Reporter.kt

```diff
@@ -85,3 +82,39 @@ private fun <T> printInColumns(groupedObjects: List<List<T>>): String {
             .map { rowCells -> rowCells.joinToString(separator = " | ", prefix = "| ", postfix = " |") }
             .joinToString(separator = "\n")
 }
+
+private class ActorWithResult(val actorRepresentation: String, val resultRepresentation: String) {
+    var spaces = 1
```

Comment:
could you make it final?

Reply:
@ndkoval, IDE says that 'final' is redundant...

Reply:
I mean the field

Reply:
I mean the field too.
Do you want to prevent possible future inheritors from overriding the field?
If not, it seems to me that it won't have any difference with `final`

Reply:
Or do you ask to replace `var` with `val`?